doc/Doxyfile.in

```diff
@@ -107,7 +107,7 @@ RECURSIVE              = YES
 EXCLUDE                = @CMAKE_DOXYGEN_EXCLUDE_LIST@
 EXCLUDE_SYMLINKS       = NO
 EXCLUDE_PATTERNS       = *.inl.hpp *.impl.hpp *_detail.hpp */cudev/**/detail/*.hpp *.m */opencl/runtime/* */legacy/* *_c.h @DOXYGEN_EXCLUDE_PATTERNS@
-EXCLUDE_SYMBOLS        = cv::DataType<*> cv::traits::* int void CV__* T __CV*
+EXCLUDE_SYMBOLS        = cv::DataType<*> cv::traits::* int void CV__* T __CV* service cv::service
```

Comment:
Lets avoid that.

Please try to use this pattern:

```
#ifdef CV_DOXYGEN
... well documented macros without implementation or with documentation friendly stuff ...
#else
... real macro implementations (excluded from documentation) ...
#endif // CV_DOXYGEN
```

or approach from core/utils/trace.hpp (basically the same)

Reply:
Applied.